client/dashboard/sites/monitoring-http-responses-card/http-codes.tsx

```diff
@@ -0,0 +1,200 @@
+import { __ } from '@wordpress/i18n';
+
+export interface HTTPCodeSerie {
+	statusCode: number;
+	label: string;
+	showInLegend?: boolean;
+}
+
+export const useSuccessHttpCodeSeries = () => {
+	const series: HTTPCodeSerie[] = [
+		{
+			statusCode: 200,
+			label: __( '200: OK Response' ),
+			showInLegend: true,
+		},
+		{
+			statusCode: 301,
+			label: __( '301: Moved Permanently' ),
+			showInLegend: true,
+		},
+		{
+			statusCode: 302,
+			label: __( '302: Moved Temporarily' ),
```

Comment:
Nit: 302 is now "Found" https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status/302